Marlin/Configuration.h

```diff
@@ -646,6 +646,8 @@ const bool Z_MIN_PROBE_ENDSTOP_INVERTING = false; // set to true to invert the l
   #define EEPROM_CHITCHAT // Please keep turned on if you can.
 #endif
 
+//#define HOST_KEEPALIVE_FEATURE // Send messages to the host when busy with long operations
```

Comment:
Well.
In that case let's go for it.
But we need at least a week for tests after merging to RCBugFix, before we can tag RC4.
The nozzle-tune disaster still hurts.